Applications/DataExplorer/DataView/MeshTabWidgetBase.ui

```diff
@@ -1,20 +1,29 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ui version="4.0">
- <class>MshTabWidgetBase</class>
- <widget class="QWidget" name="MshTabWidgetBase">
+ <class>MeshTabWidgetBase</class>
+ <widget class="QWidget" name="MeshTabWidgetBase">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>278</width>
-    <height>500</height>
+    <height>511</height>
```

Comment:
Is this size change intended?

Reply:
No idea how this happened. Some Qt-Designer magic?  
It doesn't really matter because the actual size is derived from the Data Explorer window. I'll change it back anyway...